src/Containers/LoyaltyPoints/LoyaltyPoints.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'; 
+import styled from 'styled-components';
+import { Stars } from '@styled-icons/material/Stars';
+import {Paragraph} from '@Text';
+
+export interface LoyaltyPointsProps
+    extends React.HTMLAttributes<HTMLDivElement>{
+            LoyaltyAmount: number;
```

Comment:
Why are props uppercase? With no docs

Reply:
Im having an issue with the text colour being forced to be the colour of paragraph rather than my styled div since changing to it, cant figure out how to fix that.